hera_sim/tests/test_foregrounds.py

```diff
@@ -1,220 +1,138 @@
 
-        dfft = FFT(vis * dspec.gen_window("blackmanharris", len(vis))[:, None], axis=0)
+    conj_kwargs = kwargs.copy()
+    conj_kwargs["bl_vec"] = -bl_vec
+    vis = model(**kwargs)
+    conj_vis = model(**conj_kwargs)
+    assert np.allclose(vis, conj_vis.conj())  # Assert V_ij = V*_ji
 
-        max_frate = frates[np.argmax(np.abs(dfft[:, 0]))]
-        nt.assert_true(max_frate < 0)
 
-    @pytest.mark.skip("it turns out that V_ij = V_ji for this model; how do we fix it?")
-    def test_diffuse_foreground_conjugation(self):
-        # make some parameters
-        freqs = np.linspace(0.1, 0.2, 100, endpoint=False)
-        omega_p = Beam(beamfile)
-        lsts = np.linspace(0, 2 * np.pi, 1000)
-        Tsky_mdl = noise.HERA_Tsky_mdl["xx"]
-        bl_vec = np.asarray([10, 0, 0])
-        delay_filter_kwargs = {"delay_filter_type": "tophat"}
-        fringe_filter_kwargs = {"fringe_filter_type": "tophat"}
-
-        np.random.seed(0)
-        # simulate the effect
-        vis = foregrounds.diffuse_foreground(
-            lsts,
-            freqs,
-            bl_vec,
-            Tsky_mdl=Tsky_mdl,
-            omega_p=omega_p,
-            delay_filter_kwargs=delay_filter_kwargs,
-            fringe_filter_kwargs=fringe_filter_kwargs,
+def test_diffuse_foreground_exception_no_tsky_mdl(freqs, lsts, omega_p):
+    bl_vec = [0, 0, 0]
+    with pytest.raises(ValueError) as err:
+        _ = foregrounds.diffuse_foreground(
+            freqs=freqs, lsts=lsts, bl_vec=bl_vec, Tsky_mdl=None, omega_p=omega_p,
         )
+    assert "sky temperature model must be" in err.value.args[0]
 
-        np.random.seed(0)
-        # simulate the effect
-        _vis = foregrounds.diffuse_foreground(
-            lsts,
-            freqs,
-            -bl_vec,
-            Tsky_mdl=Tsky_mdl,
-            omega_p=omega_p,
-            delay_filter_kwargs=delay_filter_kwargs,
-            fringe_filter_kwargs=fringe_filter_kwargs,
-        )
-
-        # XXX this test does not pass!
-        np.testing.assert_allclose(vis, _vis.conj())
-
-    @pytest.mark.skip("this doesn't pass either!")
-    def test_point_source_foreground_conjugation(self):
-        # make some parameters
-        freqs = np.linspace(0.1, 0.2, 100, endpoint=False)
-        lsts = np.linspace(0, 2 * np.pi, 1000)
-        bl_vec = np.asarray([300.0, 0])
-
-        np.random.seed(0)
-        # simulate the effect
-        vis = foregrounds.pntsrc_foreground(lsts, freqs, bl_vec, nsrcs=200)
 
-        np.random.seed(0)
-        # simulate for the conjugate baseline
-        _vis = foregrounds.pntsrc_foreground(lsts, freqs, -bl_vec, nsrcs=200)
-
-        # XXX this doesn't pass either!
-        np.testing.assert_allclose(vis, _vis.conj())
-
-
-if __name__ == "__main__":
-    unittest.main()
+def test_diffuse_foreground_exception_no_omega_p(freqs, lsts, Tsky_mdl):
+    bl_vec = [0, 0, 0]
+    with pytest.raises(ValueError) as err:
+        _ = foregrounds.diffuse_foreground(
```

Comment:
don't need the _ here, just call the function without saving the return value.

Reply:
Fixed all instances of something like this.